exercises.ipynb

```diff
@@ -121,19 +163,50 @@
+     ]
+    }
+   ],
    "source": [
     "# Exercise #2\n",
+    "def get_next_url(soup):\n",
+    "    next_url = soup.find('li', attrs = {'class':'next'})\n",
+    "    a = next_url.find('a')\n",
+    "    if a is not None:\n",
+    "        url = a.get('href')\n",
+    "        return url\n",
+    "    else: return None\n",
```

Comment:
Never use one line if or else statements in python!

Also, this would be nicer in css, no? `'li.next a'`